.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
```

Comment:
use global gitignore for ignore stuff of ide
that changes can be removed after global git ignore
that files not part of project